src/App.jsx

```diff
@@ -14,57 +16,81 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [active, setActive] = useState('Jam');
```

Comment:
you should use more meaningful variables names
```suggestion
  const [selectedGood, setSelectedGood] = useState('Jam');
```